plugins/loginmgr/accounts/YggTorrent.php

```diff
@@ -8,19 +8,19 @@ protected function isOK($client)
     {
         return (
             strpos($client->results, 'Ces identifiants sont invalides') === false
-            && strpos($client->results, 'Vous devez vous connecter pour télécharger un torrent') === false
+            && $client->results !== 'Vous devez vous connecter pour télécharger un torrent'
```

Comment:
Result contain this phrase and anything more? Are you sure?

Reply:
Yes, the URL https://yggtorrent.com/engine/download_torrent?id=<torrent_id> returns exactly this string if not logged (and with a valid torrent id).